### DISCUSSION TITLE
How can i emulate the toypad on my ps3

### DISCUSSION THREAD

**Discussion by FilmmakerAhmad** [Berny23/LD-ToyPad-Emulator#186] [Q&A]
pleaseeeeee help me run the toy pad emulator on my ps3 , i didnt understand how can i install the emulator raspberry pi zero w and conect it to the ps3 
thanks

**Comment by VladimirKuletski** [comment]
Where did you stuck with your progress?

**Comment by mathenyezra0-ctrl** [comment]
its been 2 years, and its still unanswered?? i was looking for the same question

**Reply by Berny23** [reply]
It works. Use single board computer and read the guide in the readme file.

**Reply by YTDan75** [reply]
when I run ssh *username*@*ip address* to do step 3 I get "ssh: connect to host *ip address* port 22: Connection refused."

I tried looking elsewhere to maybe find a different tutorial or a solution to this issue, but I just continue to run into different errors with each one.
So I figured at this point it's better to ask here instead.

**Reply by YTDan75** [reply]
It's been 3 weeks and I still haven't found a solution. Please help.

**Reply by Tivertoni** [reply]
@YTDan75 Are you using a micro-controller or a virtual maschine?

**Reply by YTDan75** [reply]
Neither. I'm using a raspberry pi zero 2 w

**Comment by YTDan75** [comment]
Following the tutorial I flashed bullseye to the sd card, enabled ssh in settings in the raspberry pi imager, put the sd card into the raspberry pi, booted it up, logged in, and when I connect my raspberry pi to my pc via the usb to usb data cable and try to run the ssh command to connect to it in the command prompt it gives me the port 22: connection refused error.

**Reply by YTDan75** [reply]
I tried using the ip scanner linked in the tutorial as well but the raspberry pi's ip doesn't seem to show up on the list of scanned ips at all.

**Reply by YTDan75** [reply]
I'm actually lost. I don't know what else to try. I feel like I tried everything.

If you guys have any possible solutions please reply. Thanks.
